db/document.go

```diff
@@ -91,6 +91,7 @@ type SyncData struct {
 
 	addedRevisionBodies     []string          // revIDs of non-winning revision bodies that have been added (and so require persistence)
 	removedRevisionBodyKeys map[string]string // keys of non-winning revisions that have been removed (and so may require deletion), indexed by revID
+	ClusterUUID             string            `json:"cluster_uuid,omitempty"`
```

Comment:
We should move this up above at least `History` in this struct - see comment explaining why.

Sidenote/thought: Is it possible to get test coverage for this "History should be marshalled last" to catch future cases? I'll file a ticket for it.